### ISSUE TITLE
Update the application to use TIRRC Form

### ISSUE THREAD

**Issue by CoreyAR** [code-for-nashville/power-of-attorney#110]
Now that we have a POC an access to the POC form used by TIRRC, we need to update the application to use this form. 

The form is pinned in the power-of-attorney channel on the Code for Nashville slack.
https://nashvillebrigade.slack.com/files/U2AK5BYNP/FH1DHFGGK/poa_sm_-_tirrc.docx

**Comment by combinatorist** [comment]
I thought this was done, @CoreyAR. Is there more work to do on this? Do you need anything from say TIRRC or Lori?

Thanks!
Tim

**Comment by CoreyAR** [comment]
Yes,  this is still in progress. You can reference progress on PR #112
